pages/common/conda.md

```diff
@@ -0,0 +1,35 @@
+# conda
+
+> Package, dependency and environment management for any programming language.
+
+- Create a new environment, installing named packages:
+
+`conda create --name {{environment_name}} {{packages}}`
+
+- List all environments:
+
+`conda info --envs`
+
+- Load or unload an environment:
+
+`source {{activate|deactivate}} {{environment_name}}`
+
+- Delete an environment (remove all packages):
+
+`conda remove --name {{environment_name}} --all`
+
+- Search conda channels for a package by name:
+
+`conda search {{package_name}}`
+
+- Install one or more packages into the current environment:
```

Comment:
NB: Haven't mentioned conda channels here at all. For example, many packages I used are part of [bioconda](https://bioconda.github.io/) and could be installed with `-c bioconda` (if the channel config isn't already saved).